src/main/java/service/impl/ReportCreatorImpl.java

```diff
@@ -0,0 +1,25 @@
+package service.impl;
+
+import static db.Storage.fruitStorage;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.ReportCreator;
+
+public class ReportCreatorImpl implements ReportCreator {
+    @Override
+    public List<String> create() {
+        List<String> report = new ArrayList<>();
+        Field[] fields = FruitTransaction.class.getDeclaredFields();
+        String column0Name = fields[1].getName();
+        String column1Name = fields[2].getName();
```

Comment:
there is no need to use reflection, you can hardcode String values according to task